src/components/DepositCrypto/index.tsx

```diff
@@ -69,7 +78,10 @@ const DepositCrypto: React.FunctionComponent<DepositCryptoProps> = (props: Depos
                     <p className="cr-deposit-info">{text}</p>
                     {data ? <div className="d-none d-md-block qr-code-wrapper"><QRCode dimensions={size} data={data}/></div> : null}
                 </div>
-                <div>
+                <div className="cr-deposit-crypto__block">
+                    {currency === 'eth' && !isMobileDevice ? (
```

Comment:
maybe it makes sense to put 'eth' to constants. It's possible that we'll support other currencies deposit from Metamask account as well.

Reply:
no need to move it to the constants because it's using only with metaMask. In near future this code will be replaced with method that check for ERC20 currency type (now blocked by lack of the data from the BE)

Reply:
Got it